include/zephyr/input/input_analog_axis.h

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_INPUT_ANALOG_AXIS_H_
+#define ZEPHYR_INCLUDE_INPUT_ANALOG_AXIS_H_
+
+#include <stdint.h>
+#include <zephyr/device.h>
+
+/**
+ * @brief Keyboard Matrix API
+ * @defgroup input_kbd_matrix Keyboard Matrix API
```

Comment:
copy-paste issue? whatever new group you end up with, it would be good to add it to `doc/services/input/index.rst`

Reply:
oof, thanks, forgot to change it